core/arch/arm/plat-vexpress/platform_config.h

```diff
@@ -221,7 +221,7 @@
 #endif
 
 #define DRAM0_BASE		UINTPTR_C(0x40000000)
-#define DRAM0_SIZE		(UINTPTR_C(0x40000000) - CFG_SHMEM_SIZE)
+#define DRAM0_SIZE		(UINTPTR_C(0x42100000) - CFG_SHMEM_SIZE)
```

Comment:
I'd rather update `build/qemu_v8.mk`.
Long term we could read (or supplement) the DRAM config from DT on the QEMU platforms.